src/plugins/intel_gpu/tests/test_cases/concatenation_gpu_test.cpp

```diff
@@ -612,6 +725,21 @@ INSTANTIATE_TEST_SUITE_P(smoke,
                         concat_gpu_all_params,
                         concat_gpu::PrintToStringParamName);
 
+TEST_P(concat_gpu_4d_axis3_f16, fs_b_yx_fsv32) {
+    ASSERT_NO_FATAL_FAILURE(test(format::fs_b_yx_fsv32));
+}
+
+INSTANTIATE_TEST_SUITE_P(smoke,
+                        concat_gpu_4d_axis3_f16,
+                        ::testing::Values(
+                            TestParamType_concat_axis3(2, 8, 2, { 2, 2 }),
+                            TestParamType_concat_axis3(2, 32, 2, { 31, 64 }),
+                            TestParamType_concat_axis3(2, 16, 2, { 15, 15, 16 }),
+                            TestParamType_concat_axis3(2, 16, 2, { 16, 15, 16 }),
+                            TestParamType_concat_axis3(2, 16, 1, { 15, 2, 16, 64 })
+                        ),
```

Comment:
To test fsv32, it would be good to test with larger **feature** size than 32.
(I think, X does not need to be that big, but larger feature size is more meaningful.) 
Also It would be good to have a test with unaligned shapes like 2,  **35**, 2, { .... }

Reply:
Done.